menas/pom.xml

```diff
@@ -196,7 +195,11 @@
             <artifactId>webjars-locator-core</artifactId>
             <version>${webjars.locator.version}</version>
         </dependency>
-
+        <dependency>
```

Comment:
This might be for bigger discussion and possibly contagious 😉 . But do we need _Oozie_ integration within _Menas_? It's becoming quite a fat baby. 🤔

Reply:
Different discussion.